azerothcore/azerothcore-wotlk
# fix(DB/Dungeon/Quest): Entry into Karazhan

## CHANGES PROPOSED:
-  Fix the spawn of the "First Fragment Guardian" for the quest Entry into Karazhan


## ISSUES ADDRESSED:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/3033



## TESTS PERFORMED:
- Spawned the creature, it spawned correctly now



## HOW TO TEST THE CHANGES:

.go gob 22674
.quest add 9831
Right click on the jar
The NPC should spawn on top of the jar, like here: https://www.youtube.com/watch?v=SHHmDpLSFZw
If you can't execute the query, keep in mind the Safe Mode of your client needs to be disabled ( as event_scripts table doesn't have any primary key )

## KNOWN ISSUES AND TODO LIST:



## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
